Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN pipenv install --deploy --system
 
 COPY . .
 
+RUN chown -R fsdrui /opt/ui/app/assets/
```

Comment:
As-is, this changes the user, but not the group. `chown -R fsdrui: ...` or `chown -R fsdrui::fsdrui ...` might be preferable, but it's a very minor point.

Reply:
Changed